MiniApp/Classes/MiniAppStorage.swift

```diff
@@ -0,0 +1,7 @@
+class MiniAppStorage {
+    internal func save(sourcePath: URL, destinationPath: URL) {
+        try? FileManager.default.createDirectory(atPath: destinationPath.relativePath, withIntermediateDirectories: true)
+        try? FileManager.default.removeItem(at: destinationPath)
+        try? FileManager.default.copyItem(at: sourcePath, to: destinationPath)
```

Comment:
Can any of these fail? Do you need to handle/report the errors?

Reply:
Updated